controllers/argocd/util_test.go

```diff
@@ -465,6 +465,48 @@ func TestGetArgoApplicationControllerCommand(t *testing.T) {
 			[]argoCDOpt{controllerProcessors(10)},
 			controllerProcesorsChangedResult("10"),
 		},
+		{
+			"configured status processors to zero",
```

Comment:
what are these unit tests testing ? It should be testing `sourceNamespaces` field right?